sql/sql_mode_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sql
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSqlMode(t *testing.T) {
+	// Test that ANSI mode includes ANSI_QUOTES mode
+	sqlMode := NewSqlModeFromString("only_full_group_by,ansi")
+	require.True(t, sqlMode.AnsiQuotes())
```

Comment:
Nit: more idiomatic to use `assert` for assertions that can allow the rest of the test continue safely